README.md

```diff
@@ -35,6 +35,11 @@ distro packages (deb, rpm, etc.) that have all dependencies bundled up together
 installation with your package manager, but with as a hermetically sealed app
 that doesn't depend on the rest of your distro.
 
+For Ubuntu users, Sugarjar will likely be available starting in the upcoming
```

Comment:
Move this above the omnibus section please

Reply:
done. also listed the Ubuntu versions supported